loader/loader.js

```diff
@@ -14,13 +14,13 @@
 * limitations under the License.
 *******************************************************************************/
 
-module.exports = function (loadUtil,settings) {
+module.exports = function (loadUtil, settings) {
     var module = {};
     
 	var csv = require('csv');
 	var log4js = require('log4js');
 	var uuid = require('node-uuid');
-	var async = require('async');
+    const async = require('./async')
```

Comment:
Maybe consider renaming this const?  Although async is not a reserved word, the usage of this const can be confusing to read.

Reply:
A valid point. I left `async` so we keep the method signature and make the delta minimal.